src/Main.java

```diff
@@ -0,0 +1,363 @@
+                }
+            }
+        }
+        return maxElement;
+    }
+
+    public static int[][] swapMinAndMaxElementsInTwoDimensionalArray (int[][] givenArray) {
+        int minElement = findMinElement(givenArray);
+        int maxElement = findMaxElement(givenArray);
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                if (givenArray[i][j] == minElement) {
+                    givenArray[i][j] = maxElement;
+                }
+                else if (givenArray[i][j] == maxElement) {
+                    givenArray[i][j] = minElement;
+                }
+            }
+        }
+        return givenArray;
+    }
+
+    public static int[][] swapMinAndMaxElementsInTwoDimensionalArray2 (int[][] givenArray) {
+        int minElement = givenArray[0][0];
+        int maxElement = givenArray[0][0];
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                if (givenArray[i][j] < minElement) {
+                    minElement = givenArray[i][j];
+                }
+                else if (givenArray[i][j] > maxElement) {
+                    maxElement = givenArray[i][j];
+                }
+            }
+        }
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                if (givenArray[i][j] == minElement) {
+                    givenArray[i][j] = maxElement;
+                }
+                else if (givenArray[i][j] == maxElement) {
+                    givenArray[i][j] = minElement;
+                }
+            }
+        }
+        return givenArray;
+
+    }
+
+    public static int[][] replaceOddOnEvenElementInTwoDimensionalArray (int[][] givenArray) {
+        printTwoDimensionalArray(givenArray);
+        System.out.println();
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                if (givenArray[i][j] % 2 != 0 && j != 0) {
+                    givenArray[i][j] = givenArray[i][j-1];
+                }
+                else if (givenArray[i][j] % 2 != 0 && j == 0) {
+                    givenArray[i][j] = 0;
+                }
+            }
+        }
+        return givenArray;
+    }
+
+    public static int multiplyDiagonalElementsRelativeEnteredElement (int[][] givenArray, int number) throws NoElementInArray {
+        int multiply = 0;
+        int indexOfLine = -1;
+        int indexOfElementInLine = -1;
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                if (givenArray[i][j] == number) {
+                    indexOfLine = i;
+                    indexOfElementInLine = j;
+                    break;
+                }
+            }
+        }
+        System.out.println("indexOfLine " + indexOfLine);
```

Comment:
don't put all this trash into your code) delete all unnecessary printing

Reply:
deleted